### ISSUE TITLE
Export - Detect already exported file using only the filename -- ignore the filetype

### ISSUE THREAD

**Issue by JimShip** [jessedp/tablo-tools-electron#110]
I routinely export movies and shows and then follow up by recoding the H-264 mp4 to H-265 mkv format to save space.
This breaks the test for a file that has already been exported.

Options:
1) A simple solution would be to simply ignore the filetype as the show/movie is fully defined by its filename and the filetype is irrelevant.

2) I could simply always use the mp4 container for my recoded file, but it seems cleaner to just ignore the filetype.

3) Including in the export setup section a button to ignore the filetype would work also without breaking other people's setups.

4) Allowing a template similar to the export naming templates would give complete control over what is considered an already exported file.

Thanks,
Jim Shipman

**Comment by jessedp** [comment]
Thank you for the feedback. I tagged this and will leave it open in the event I cobble together a new version.

**Comment by ChuckDobbs** [comment]
I'm seeing some strange behavior that may be related. My Assumption is if a show/movie has been exported you can set the export contingent on whether a file with the same name exists in the target directory.  i.e., skip, overwrite, or append...  This functionality is not working for me. The file simply overwrites the older file on in the target directory in every case. I tried exporting a movie a second time directly after.  I am,  however,  seeing potential export filenames color coded but not always correctly.  This ticket makes it sound like this feature works somewhere. Is this related in any way or should i open a separate ticket? (Or am i misunderstanding the intended behavior?)

I just recently loaded the latest release. I like what i see so far. If i can be of any assistance, please let me know. 

Thanks


![Export1](https://user-images.githubusercontent.com/112043469/186563653-60ddb5b5-bcb3-44b0-88f4-c40c6e5ef708.png)
